egs/wsj/s5/steps/nnet3/train_raw_dnn.py

```diff
@@ -286,7 +288,9 @@ def train(args, run_opts):
     # use during decoding
     common_train_lib.copy_egs_properties_to_exp_dir(egs_dir, args.dir)
 
-    if (args.stage <= -3) and os.path.exists(args.dir+"/configs/init.config"):
+    add_lda = common_train_lib.is_lda_added(config_dir)
+
```

Comment:
did you see my comment:
"I think you are mistaken, if we are talking about the current kaldi_52
code; init.config is only used if we are doing the LDA thing."
can you please update the PR?